src/MatBlazor.Demo/Demo/DemoMatSlider.razor

```diff
@@ -283,3 +283,51 @@
     ")></BlazorFiddle>
     </SourceContent>
 </DemoContainer>
+
+<h5 class="mat-h5">Value update</h5>
+<DemoContainer>
+    <Content>
+        <MatTextField @bind-Value="@Value" Label="Value"></MatTextField>
+        <MatTextField @bind-Value="@ValueMin" Label="ValueMin"></MatTextField>
+        <MatTextField @bind-Value="@ValueMax" Label="ValueMax"></MatTextField>
+        <MatSlider TValue="decimal" Value="@Value" ValueMin="@ValueMin" ValueMax="@ValueMax" ValueChanged="@OnValueChanged2"></MatSlider>
+
+        @code
+        {
+
+            public decimal Value { get; set; } = 50;
+            public decimal ValueMin { get; set; } = 0;
+            public decimal ValueMax { get; set; } = 100;
+
+            public void OnValueChanged2(decimal val)
```

Comment:
This method is highly ambiguous to any dev coming into the code. It would be cleaner to help clean up the name of the method. Likewise, since this is a one-liner, you could use an expression body method instead for conciseness.

Reply:
actually there already is a method named **OnValuChanged** on L261 so i cannot use that. i cannot possible image better method name in this case since there is a value being changed - nothing more. and i believe that dev would understand that it is a demo page with multiple examples of a given component so i dont personally consider number suffix as a big deal. furthermore, when there are multiple variables named Val, Val2 - Val5 in the same file. i agree that it could be expression bodied method but i wanted to follow existing format.